pkg/graph/schema.resolvers.go

```diff
@@ -49,6 +49,23 @@ func (r *queryResolver) AccessibilityRequest(ctx context.Context, id uuid.UUID)
 	return r.store.FetchAccessibilityRequestByID(ctx, id)
 }
 
+func (r *queryResolver) SystemsList(ctx context.Context) ([]*model.System, error) {
+	items, err := r.systemsList(ctx)
+	if err != nil {
+		return nil, err
+	}
+	var results []*model.System
+	for _, item := range items {
+		results = append(results, &model.System{
+			IntakeID:    &item.IntakeID,
+			LifecycleID: &item.LCID,
+			ProjectName: &item.ProjectName,
+			OwnerName:   &item.OwnerName,
+		})
+	}
+	return results, nil
+}
+
```

Comment:
Translating between `models` and `generated/model` types feels like a busywork sink... I know @Igarfinkle found some docs somewhere about using struct tags to help ease the Go->GraphQL marshaling, but on a quick scan I didn't see where `gqlgen` provided capabilities for that.

Reply:
+1 to sorting this out and avoiding busywork!

Reply:
Yes, we can use struct tags! 

Specifically, on my forthcoming, I turned on the use of a struct tag in gqlgen.yml 

```# Optional: turn on use `gqlgen:"fieldName"` tags in your models
struct_tag: gqlgen```

So then in a model, a field tagged `gqlgen: "name_in_schema"` will be automatically associated to that name in the schema

Reply:
Ping me if you want to talk through it